src/state/models/feeds/posts.ts

```diff
@@ -47,6 +52,7 @@ export class PostsFeedModel {
   isBlockedBy = false
   error = ''
   loadMoreError = ''
+  cleanError?: PostsFeedModelError = undefined
```

Comment:
One idea for attaching more info to the error so we know more about what happened